scala/build/Dockerfile.latest

```
@@ -0,0 +1,87 @@
+FROM ubuntu:oracular
+
+# Maintainer information
+LABEL maintainer="me@lehungio.com"
+
+# Set environment variables
+ENV DEBIAN_FRONTEND=noninteractive \
+PATH="/home/qa/.yarn/bin:/home/qa/.local/bin:${PATH}" \
+QA_PATH="/home/qa/code"
+
+# Update package index and install system dependencies
+RUN apt-get update && apt-get install -y \
```

Hadolint

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)